workspaces/variables.md

```diff
@@ -43,6 +43,10 @@ env | grep CODER_
         <td><code>CODER_IMAGE_URI</code></td>
         <td>The URI of the image used to build the workspace</td>
     </tr>
+    <tr>
+        <td><code>CODER_MAX_WORKSPACES_PER_USER</code></td>
+        <td>The maximum number fo workspaces allowed per user</td>
+    </tr>
```

Comment:
```suggestion
```

This actually isn't an env we inject into workspaces, it's an env that the admin can supply to coderd through the new `coderd.extraEnvs` value to control the max amount of workspaces. We probably need a new documentation page somewhere for env variables that coderd uses, but that can happen later. For now we can just tell the customer that requested this how to use it directly.